include/seqcmd.h

```diff
@@ -0,0 +1,510 @@
+#ifndef SEQCMD_H
+#define SEQCMD_H
+
+// ==== Primary commands ====
+
+#define SEQCMD_MASK 0xF0000000
+
+typedef enum {
+    /* 0x0 */ SEQCMD_OP_PLAY_SEQUENCE,
+    /* 0x1 */ SEQCMD_OP_STOP_SEQUENCE,
+    /* 0x2 */ SEQCMD_OP_QUEUE_SEQUENCE,
+    /* 0x3 */ SEQCMD_OP_UNQUEUE_SEQUENCE,
+    /* 0x4 */ SEQCMD_OP_SET_PLAYER_VOLUME,
+    /* 0x5 */ SEQCMD_OP_SET_PLAYER_FREQ,
+    /* 0x6 */ SEQCMD_OP_SET_CHANNEL_VOLUME,
+    /* 0x7 */ SEQCMD_OP_SET_PLAYER_IO,
+    /* 0x8 */ SEQCMD_OP_SET_CHANNEL_IO,
+    /* 0x9 */ SEQCMD_OP_SET_CHANNEL_IO_DISABLE_MASK,
+    /* 0xA */ SEQCMD_OP_SET_CHANNEL_DISABLE_MASK,
+    /* 0xB */ SEQCMD_OP_TEMPO_CMD,
+    /* 0xC */ SEQCMD_OP_SETUP_CMD,
+    /* 0xD */ SEQCMD_OP_SET_CHANNEL_FREQ,
+    /* 0xE */ SEQCMD_OP_GLOBAL_CMD,
+    /* 0xF */ SEQCMD_OP_RESET_AUDIO_HEAP
+} SeqCmdType;
+
+// ==== Secondary commands ====
+
+// Subset of `SEQCMD_OP_TEMPO_CMD`
+typedef enum {
+    /* 0x0 */ SEQCMD_SUB_OP_TEMPO_SET,
+    /* 0x1 */ SEQCMD_SUB_OP_TEMPO_SPEED_UP,
+    /* 0x2 */ SEQCMD_SUB_OP_TEMPO_SLOW_DOWN,
+    /* 0x3 */ SEQCMD_SUB_OP_TEMPO_SCALE,
+    /* 0x4 */ SEQCMD_SUB_OP_TEMPO_RESET
+} TempoCmdType;
```

Comment:
I feel like naming these `SeqCmdXxxType` or something like that would be less prone to overlapping names? (have "seq cmd" in there)